.gitignore

```diff
@@ -1,13 +1,14 @@
 # Ignore directories generated by Composer
 drush/contrib
 vendor
-web/core
-web/modules/contrib
-web/themes/contrib
-web/profiles/contrib
+app/core
```

Comment:
bikeshed: app is very generic, why not just use "drupal" ? In case you have some other stuff like the mentioned pattern-lab in the repo also, that makes things clear.